tests/version.bats

```diff
@@ -11,12 +11,12 @@ load helpers
 @test "buildah version current in .spec file Version" {
 	bversion=$(buildah version | awk '/^Version:/ { print $NF }')
 	rversion=$(cat ${TESTSDIR}/../contrib/rpm/buildah.spec | awk '/^Version:/ { print $NF }')
-	run test "${bversion}" = "${rversion}"
+	run test "${bversion}" = "${rversion}" -o "${bversion}" = "${rversion}-dev"
 	[ "$status" -eq 0 ]
 }
 
 @test "buildah version current in .spec file changelog" {
 	bversion=$(buildah version | awk '/^Version:/ { print $NF }')
-	run bash -c "grep -A1 ^%changelog ${TESTSDIR}/../contrib/rpm/buildah.spec | grep -q \" ${bversion}-.*$\""
+	run bash -c "grep -A1 ^%changelog ${TESTSDIR}/../contrib/rpm/buildah.spec | grep -q \" ${bversion}.*$\""
```

Comment:
Can I nit, and ask that this be reverted? ${bversion}- is correct, in this case 1.2-dev with -1. This was to stop people just pasting in the version number, when you should also include the package release number. I can revert it after the commit as I know this commit is something you want to get through.